packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/en.ftl

```diff
@@ -6,6 +6,7 @@ complete-reset-password-warning-message-2 = <span>Remember:</span> When you rese
 # A new password was successfully set for the user's account
 # Displayed in an alert bar
 complete-reset-password-success-alert = Password set
-# An error occured while attempting to set a new password (password reset flow)
+# An error occurred while attempting to set a new password (password reset flow)
 # Displayed in an alert bar
 complete-reset-password-error-alert = Sorry, there was a problem setting your password
+complete-reset-password-recovery-key-error = Sorry, there was a problem checking if you have an account recovery key. If you do, <hasRecoveryKeyErrorLink>click here to reset your password with it</hasRecoveryKeyErrorLink>.
```

Comment:
```suggestion
complete-reset-password-recovery-key-error = Sorry, there was a problem checking if you have an account recovery key. If you have one, <hasRecoveryKeyErrorLink>click here to reset your password with it</hasRecoveryKeyErrorLink>.
```
For some reason my brain had trouble parsing what `If you do` was referring to, so adding a suggestion to improve ease of understanding.

Reply:
I could almost argue "If you have one, click here to reset your password with it." could maybe be less-ambiguously rewritten as:
```jsx
<hasRecoveryKeyErrorLink>Reset your password with your account recovery key.</hasRecoveryKeyErrorLink>
```

Reply:
I'm going to go with your suggestion @pdehaan, Vesta told me we're probably taking a closer look at these pages soon and I'm the one that suggested the original copy.